x-pack/legacy/plugins/uptime/public/components/functional/filter_group/filter_group.tsx

```diff
@@ -124,7 +152,66 @@ export const FilterGroupComponent = ({
   );
 };
 
-export const FilterGroup = withUptimeGraphQL<FilterBarQueryResult, FilterBarDropdownsProps>(
-  FilterGroupComponent,
-  filterBarQuery
-);
+const filterWhitelist = [
```

Comment:
I feel like we could move this to be defined in the same file as `parseFiltersMap` and not pass this in as a parameter. We don't have a use case yet where we need to pass in and parse multiple field definitions, so why add the extra parameter with all the cognitive load that follows?

Reply:
Yeah sure, we should do that. Added here: https://github.com/elastic/kibana/pull/50837/commits/f13d97c10adea7821353ed5c5e8cc5ba16a6ae40 along with a comment explaining the constant value and prompting future maintainers to add a parameter if they need to enhance the functionality.